lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+require 'dish'
+
+class Restaurant
+  attr_accessor :dishes
+  
+  def initialize
+    self.dishes = []
+  end
+  
+  def new_dish(name, price)
+    dish = Dish.new(name: name, price: price)
+    self.dishes.push dish
+  end
+  
+  def send_notification(reciepient)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `reciepient`. If it's necessary, use `_` or `_reciepient` as an argument name to indicate that it won't be used. You can also write as `send_notification(*)` if you want the method to accept any arguments but don't care about them.